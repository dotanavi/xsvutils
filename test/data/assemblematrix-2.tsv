word	week	score1	hit
A	2018-07-02	100	3
B	2018-07-02	80	8
C	2018-07-02	120	100
B	2018-07-09	110	9
C	2018-07-09	130	120
D	2018-07-09	50	2100
