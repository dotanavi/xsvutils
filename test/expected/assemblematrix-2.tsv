word	2018-07-02-score1	2018-07-02-hit	2018-07-09-score1	2018-07-09-hit
A	100	3		
B	80	8	110	9
C	120	100	130	120
D			50	2100
